riak_test/log_handoff_test.erl

```diff
@@ -75,6 +75,7 @@ test_handoff(RootNode, NewNode, NTestItems) ->
     %% added to the root node.  Note: systest_read() returns
     %% /non-matching/ items, so getting nothing back is good:
     lager:info("Validating data after handoff:"),
+    timer:sleep(100),
```

Comment:
The test passed on my local machine, but failed in Jenkins. So I guess that the problem is because vectorclock service is not started yet when the query to riak_core_apl:active_owners(vectorclock) is done, so I added this line to check and the test passed on Jenkins. Do you know any better way to solve this?